services/post_service.py

```
@@ -0,0 +1,112 @@
+
+            if author_info:
+                post["author"] = {
+                    "id": encode_id(author_info.get("id")),
+                    "nickname": author_info.get("nickname"),
+                }
+            #좋아요의 기본값은 false
+            post["is_liked"] = False
+
+        return paged_post, len(all_posts)
+
+    async def get_posts_detail(self,post_id:int,user_id:str | None = None) -> dict:
```

Ruff (0.14.13)

Unused method argument: `user_id`

(ARG002)